filterable-file-log/serializer.lua

```diff
@@ -60,11 +64,46 @@ function _M.serialize(ngx, filters)
     service = ngx.ctx.service,
     api = ngx.ctx.api,
     consumer = ngx.ctx.authenticated_consumer,
+    jwt_claims = jwt_claims,
     client_ip = ngx.var.remote_addr,
     started_at = ngx.req.start_time() * 1000
   }
 end
 
+function get_jwt_claims(headers)
+  if not headers['authorization'] then
+    return nil
+  end
+
+  local token = string.match(headers['authorization'], 'Bearer (.+)')
+  if not token then
+    return nil
+  end
```

Comment:
Maybe this regex should be used to limit the captured content to a rough JWT-looking value?